Validation/Editor/ValidationUtil.cs

```diff
@@ -76,7 +82,19 @@ public static class ValidationUtil {
 			});
 		}
 
-		public static IList<IValidationError> ValidateAllGameObjectsInScenes(IEnumerable<Scene> scenes, bool earlyExitOnError = false) {
+		public static bool HasToPersistScene()
+		{
+			if (UnityEditorInternal.InternalEditorUtility.inBatchMode)
```

Comment:
Seems like we can use `Application.isBatchMode` instead of internal API right?

https://docs.unity3d.com/ScriptReference/Application-isBatchMode.html

Reply:
Good catch. I'll change it 😄